src/panels/config/js/trigger/device.tsx

```diff
@@ -41,6 +90,19 @@ export default class DeviceTrigger extends Component<any, any> {
           hass={hass}
           label="Trigger"
         />
+        <paper-time-input
```

Comment:
tslint complains: `Property 'paper-time-input' does not exist on type 'JSX.IntrinsicElements'.`

Reply:
See `preact-types.ts`

Reply:
Fixed.